README.md

```diff
@@ -42,6 +42,21 @@ Committing to `master` requires a code review, submitted in the form of a GitHub
 have an explicit approval from a core team member and no outstanding questions.
 
 
+## Application
```

Comment:
**Application** doesn't seem as descriptive as the other headings in the document when I scan it in context. Maybe rename this section to e.g. **Getting Started** or **Tool Support**?